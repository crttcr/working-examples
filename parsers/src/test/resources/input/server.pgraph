// Example server resources
//
//
## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix    = xiv.server

(s1: SERVER { name = "alpha", ip="10.0.2.101"})
(s2: SERVER { name = "bogon", ip="10.0.2.102"})
