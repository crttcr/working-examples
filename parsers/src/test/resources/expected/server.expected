## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix = xiv.server
VERTEX: s1
	LABEL: SERVER
		PROP name = "alpha"
		PROP ip = "10.0.2.101"
VERTEX: s2
	LABEL: SERVER
		PROP name = "bogon"
		PROP ip = "10.0.2.102"
